root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -34,9 +34,13 @@ MB.Control.initialize_guess_case = function (type, formPrefix) {
     }
 
     $name.parent()
-        .find("button.guesscase-title").on("click", function () { setVal($name, guess.guess($name.val())) })
+        .find("button.guesscase-title").on("click", function () {
```

Comment:
Mind starting `.on` on the next line (at the same indentation level as the other method calls) so it doesn't conflict with #1311? :)